src/util/BaseRestClient.ts

```diff
@@ -115,6 +115,8 @@ export default abstract class BaseRestClient {
       sync_interval_ms: 3600000,
       /** Request parameter values are now URI encoded by default during signing. Set to false to override this behaviour. */
       encodeSerialisedValues: true,
+      /** Force sync time before send private request */
+      forceSync: false,
```

Comment:
I'm OK with this kind of optional feature, though I'd like to give it a self explanatory name:
```suggestion
      timeSyncBeforeRequest: false,
```

Reply:
In addition to the nitpicky rename of this new property, can you also add the boolean to the typescript interface for the REST options object? This is necessary to make it available to anyone building in typescript:
https://github.com/tiagosiebler/bybit-api/blob/master/src/util/requestUtils.ts#L1-L43

Reply:
I changed the var name, thx for reviewing